### PR TITLE
feat: sponsor section added

### PR DESCRIPTION
**Pull request by Rupantar-Borphukan**
**sponsor section and fav icon is added.**

sponser_section

### CODE REVIEW

**Comment by codadept** [comment]
image

At first glance, there are many issues

- Use relative units and don't use absolute units
- Fix the horizontal scroll bar

File: src/assets/Images/amul_logo.png
Comment (codadept) [inline src/assets/Images/amul_logo.png:1]: Move all the images to `pubilc` folder and use the images as public assets. [Ref.](https://vitejs.dev/guide/assets#the-public-directory)

File: src/components/Sponsors/Sponsors.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import styles from '../Sponsors/Sponsors.module.css';
+
+import StarBg from "../../assets/Images/bg_landing_stars.svg";
+import Sponsor from '../../assets/Images/sponsor_logo.png';
+import frame from '../../assets/Images/frame.png';
```
Comment (codadept) [inline src/components/Sponsors/Sponsors.jsx]: After moving the images to public directory, you won't need these imports

File: src/components/Sponsors/Sponsors.module.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    top: 2.8rem;
+    float: left;
+}
+.fr_container{
+    position: relative;
+    float: left;
+    width: 8rem;
+    height: 5rem;
+    left: -3rem;
+    top: 8rem;
```
Comment (codadept) [inline src/components/Sponsors/Sponsors.module.css]: Preferred to use % or view port lengths.

**Review by codadept — CHANGES_REQUESTED** [review]

File: src/components/Sponsors/Sponsors.jsx (hunk 1 above)
Comment (Rupantar-Borphukan) [reply src/components/Sponsors/Sponsors.jsx]: Done

File: src/components/Sponsors/Sponsors.module.css (hunk 2 above)
Comment (Rupantar-Borphukan) [reply src/components/Sponsors/Sponsors.module.css]: Doing

**Comment by Rupantar-Borphukan** [comment]
- updated the sponsor_section branch
changes :   
- [ Done ] fixed horizontal scroll bar
- [ Done ] moved all img to public folder 
- [ Not yet done] responsive for mobile & tablet

File: src/components/Sponsors/Sponsors.module.css (hunk 2 above)
Comment (codadept) [reply src/components/Sponsors/Sponsors.module.css]: @heyymohit pick this up and continue making the sponsor section responsive

Comment (heyymohit) [reply src/components/Sponsors/Sponsors.module.css]: Done bhaiya

**Comment by codadept** [comment]
image

The sponsor part is not in the middle of the page. Centre it. Also use flexbox this will help it a lot. Change it to flexbox, learn about it and implement it soon.

**Review by codadept — CHANGES_REQUESTED** [review]
image

The sponsor part is not in the middle of the page. Centre it. Also use flexbox this will help it a lot. Change it to flexbox, learn about it and implement it soon.

File: src/components/Sponsors/Sponsors.jsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+const Sponsors = () => {
+  return (
+    <section className={styles.main_container}>
+      <img
+        className={styles.bg_container}
+        src="images/bg_landing_stars.svg"
+        alt="Background stars"
+      />
+      <img className={styles.text_container} src="images/sponsor_logo.png" />
```
Comment (codadept) [inline src/components/Sponsors/Sponsors.jsx]: Why is this an image??? The font is already imported in App.css

**Review by codadept — CHANGES_REQUESTED** [review]

File: src/components/Sponsors/Sponsors.jsx (hunk 3 above)
Comment (heyymohit) [reply src/components/Sponsors/Sponsors.jsx]: Rupantar used it as image. Changed it to text

**Review by codadept — APPROVED** [review]

**Pull request merged**